lib/Basic/SourceManager.cpp

```diff
@@ -2082,7 +2082,8 @@ bool SourceManager::isBeforeInTranslationUnit(SourceLocation LHS,
       return LIsScratch;
     return LOffs.second < ROffs.second;
   }
-  llvm_unreachable("Unsortable locations found");
+  //llvm_unreachable("Unsortable locations found");
+  return LB < RB;
```

Comment:
What is this change for (does not look like a debugging change)?

Reply:
Above, in `IsStructurallyEquivalent` we dump the two declarations which are not in the same redecl chain but should be.
During this dump sometimes we reached the llvm_unreachable point, because some locations are not sortable in the "To" context (which could be a result of some other import error).

Reply:
OK but a comment (FIXME) could be there to notice this. Or we should not forget to restore the code if the debug message is removed.